adding new roles to dispatcher

djdanielsson opened the pull request:
<!-- markdownlint-disable -->
# What does this PR do?
<!--- Brief explanation of the code or documentation change you've made -->
adds the new roles to dispatch

# How should this be tested?
<!--- Automated tests are preferred, but not always doable - especially for infrastructure. Include commands to run your new feature, and also post-run commands to validate that it worked. (please use code blocks to format code samples) -->
manually

# Is there a relevant Issue open for this?
<!--- Provide a link to any open issues that describe the problem you are solving. -->
no

# Other Relevant info, PRs, etc
<!--- Please provide link to other PRs that may be related (blocking, resolves, etc. etc.) -->
none

djdanielsson committed:
modified:   README.md
	new file:   changelogs/fragments/new_dispatch_roles.yml
	modified:   roles/dispatch/defaults/main.yml

Tompage1994 reviewed (changes requested):
Just wondering whether team assignments should come after user assignments?

Also, I think this value is documented in the readme as well, so probably needs updating

djdanielsson committed:
modified:   roles/dispatch/README.md
	modified:   roles/dispatch/defaults/main.yml

djdanielsson committed:
modified:   roles/dispatch/meta/argument_specs.yml

djdanielsson committed:
modified:   roles/dispatch/README.md
	modified:   roles/dispatch/defaults/main.yml
	modified:   roles/dispatch/meta/argument_specs.yml
